src/network/socketHandlers.js

```
@@ -0,0 +1,291 @@
+        attackStartTime: attackStartTime,
+        attackStartRotation: attackStartRotation,
+      };
+    } else {
+      // New player
+      players[id] = serverPlayer;
+    }
+  });
+
+  // Update our reference to myPlayer
+  setMyPlayer(players[socket.id]);
+  lastServerSync = Date.now();
```

Biome (1.9.4)

[error] The imported variable lastServerSync is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)